src/Adapter/SymfonyLoadEnv.php

```diff
@@ -9,10 +9,8 @@ class SymfonyLoadEnv implements DotEnvVars
 {
     public function exposeToEnvironment(string $dotEnvFile): void
     {
-        if (file_exists($dotEnvFile)) {
```

Comment:
> see [symfony.com/doc/4.2/components/dotenv.html#usage](https://symfony.com/doc/4.2/components/dotenv.html#usage)

The documentation is unspecific about that, but when looking at the [code](https://github.com/symfony/symfony/blob/2994ef0e1a206c485d153b8ffac881dda63bcf89/src/Symfony/Component/Dotenv/Dotenv.php#L113-L117) reveals that either one of `.env` or `.env.dist` **MUST** exist.

Therfore just removing the condition will lead to an exception, which isn't how this adapter is supposed to work.

I suggest to early return when neither file exists insead

Reply:
I went with `is_file`, because symfony uses it too in `loadEnv` - so don't bother file-system with "heavier" `file_exists`.